docs/getting_started.rst

```diff
@@ -22,16 +22,15 @@ Each Karton subsystem needs a ``karton.ini`` file that contains the connection p
 
 You can also use this file to store custom fields and use them e.g. by :ref:`example-overriding-config`.
```

Comment:
![image](https://user-images.githubusercontent.com/2192416/181044924-4aab0541-9e14-4ca3-803a-669013ac6081.png)

CI seems angry about this line :thinking:

Reply:
Fixed